ompi/communicator/comm_cid.c

```diff
@@ -395,6 +397,11 @@ static int ompi_comm_ext_cid_new_block (ompi_communicator_t *newcomm, ompi_commu
         goto fn_exit;
     }
 
+    if (!cid_base_set) {
+        opal_output_verbose (0, 0, "cid_base value is not set\n");
```

Comment:
There should not be a trailing `\n` in `opal_output_verbose()` -- it will line terminate for you.

That being said, I think we should probably `opal_show_help()` here, not just a single line output.  One of the big reasons for this is that `opal_show_help()` output is de-duplicated back in `mpirun` output (e.g., if 10,000 MPI processes emit this message, it will only be displayed once with an indication that 9,999 other processes also emitted this message).  There should probably be an appropriate help message that "This should probably never happen ... etc."